.gitignore

```diff
@@ -32,6 +32,9 @@ tsconfig.tsbuildinfo
 .idea
 .history
 
+#asdf
```

Comment:
?

Reply:
it's a command line tool used to manage other tools versions, such as nvm but for lots of tools rather than for node only: https://asdf-vm.com/guide/getting-started.html